vendor/github.com/intel/govmm/qemu/qmp.go

```diff
@@ -779,16 +779,22 @@ func (q *QMP) ExecuteVFIODeviceAdd(ctx context.Context, devID, bdf string) error
 // ExecuteVFIODeviceAdd this function receives the bus and the device address on its parent bus.
 // bus is optional. devID is the id of the device to add.Must be valid QMP identifier. bdf is the
 // PCI bus-device-function of the pci device.
-func (q *QMP) ExecutePCIVFIODeviceAdd(ctx context.Context, devID, bdf, addr, bus string) error {
+func (q *QMP) ExecutePCIVFIODeviceAdd(ctx context.Context, devID, bdf, addr, bus, sysfsdev string) error {
```

Comment:
Please submit a PR on the govmm repository, so that it can be properly reviewed :)

Reply:
Thanks, I have submitted another PR on the govmm repository.

Reply:
Could you add the link to it here so we can keep an eye on it? :)

Reply:
https://github.com/intel/govmm/pull/30